app/scripts/modules/netflix/fastProperties/domain/property.domain.ts

```diff
@@ -1,6 +1,8 @@
 import {IPlatformProperty} from './platformProperty.model';
+import {Scope} from './scope.domain';
 
 export class Property {
+  [key: string]: any;
```

Comment:
You still need this. You don't need to reference properties on class objects using bracket notation, i.e. `property['foo']` can be replaced with `property.foo`